### PR TITLE
remove allocations chart + human readable names

### PR DESCRIPTION
**Pull request by Sarahb-da**

### CODE REVIEW

File: ui/src/components/common/utils.ts [hunk 1]
```diff
@@ -119,3 +119,14 @@ export function useDismissableElement<T extends HTMLElement, C extends HTMLEleme
...
+
+export type IPartyInfo = {
+    party: string;
+    label: string;
+}
+
+export function getPartyLabel(partyId: string, parties: IPartyInfo[]) {
+    const partyInfo = parties.find(p => p.party === partyId)
+    console.log(partyInfo?.label)
+    return { label: partyInfo?.party || partyId , party: partyInfo?.label.substring(partyInfo.label.lastIndexOf('|')+1) || partyId}
```
Comment (alexmatson-da) [inline ui/src/components/common/utils.ts:131]: I'm a bit confused by this... what's an example of what a `IPartyInfo.label` looks like? Is it just the party name or that string that looks like `"name (party) | Broker"`? 

Also, why is the `partyInfo.party` assigned to the `label` field in the return here?

**Review by alexgraham-da — APPROVED** [review]
looks good to me 👍

File: ui/src/components/common/utils.ts (hunk 1 above)
Comment (Sarahb-da) [reply ui/src/components/common/utils.ts]: forgot to remove log, and switched the label-party. Copy paste error

**Review by alexmatson-da — APPROVED** [review]

File: ui/src/components/common/utils.ts (hunk 1 above)
Comment (Sarahb-da) [reply ui/src/components/common/utils.ts]: the label contains some extra information i.e. `| Broker` - if its there, I remove it here. We can remove it all together as well but the extra info is used in some places i believe

**Pull request merged**